README.md

```diff
@@ -13,21 +13,6 @@ Some GraphQL APIs have disabled introspection. For example, [Apollo Server disab
 
 Clairvoyance helps to obtain GraphQL API schema even if the introspection is disabled. It produces schema in JSON format suitable for other tools like [GraphQL Voyager](https://github.com/APIs-guru/graphql-voyager), [InQL](https://github.com/doyensec/inql) or [graphql-path-enum](https://gitlab.com/dee-see/graphql-path-enum).
 
-## Contributors
-
-Thanks to the contributors for their work.
-
-- [nikitastupin](https://github.com/nikitastupin)
-- [Escape](https://escape.tech) team
-  - [iCarossio](https://github.com/iCarossio)
-  - [Swan](https://github.com/c3b5aw)
-  - [QuentinN42](https://github.com/QuentinN42)
-  - [Nohehf](https://github.com/Nohehf)
-- [i-tsaturov](https://github.com/i-tsaturov)
-- [EONRaider](https://github.com/EONRaider)
-- [noraj](https://github.com/noraj)
-- [belane](https://github.com/belane)
-
```

Comment:
I'd keep this section and add you if you don't mind :) Thoughts?

Reply:
What if amount of contributors will raise to e.g. 30? or even 50? As an alternative we can keep them in a different file and just refer to it from the Readme. I want README to be as useful as possible so users looking at it in about one minute could understand:
1. What's this project, it's purpose.
2. How to use the project

Contributors and other details are secondary from user perspective. All the major/popular repos seem to follow this idea: user first.

Reply:
@nikitastupin I guess now it's resolved? :)